### ISSUE TITLE
Uncatched error if no "ignoredPaths" string in config.json

### ISSUE THREAD

**Issue by commanderpyle** [Krillle/homebridge-signalk#7]
This error pops up:

```

[12/10/2020, 11:23:53 AM] Homebridge v1.2.4 is running on port 52414.
[12/10/2020, 11:23:53 AM] [SignalK] websocket Subscription message sent
[12/10/2020, 11:23:53 AM] [SignalK] websocket Welcome message recieved
[12/10/2020, 11:23:53 AM] [SignalK] Adding electrical controls (EmpirBus NXT and Venus GX)
[12/10/2020, 11:23:53 AM] [SignalK] Done
[12/10/2020, 11:23:53 AM] [SignalK] Adding environment temperature and humidity
[12/10/2020, 11:23:35 AM] TypeError: Cannot read property 'indexOf' of undefined
at SignalKPlatform.noignoredPath (/homebridge/node_modules/homebridge-signalk/index.js:1042:35)
at /homebridge/node_modules/homebridge-signalk/index.js:920:19
at Array.forEach (<anonymous>)
at SignalKPlatform.processFullTree (/homebridge/node_modules/homebridge-signalk/index.js:916:16)
at Request._callback (/homebridge/node_modules/homebridge-signalk/index.js:858:20)
at Request.self.callback (/homebridge/node_modules/homebridge-signalk/node_modules/request/request.js:185:22)
at Request.emit (events.js:315:20)
at Request.<anonymous> (/homebridge/node_modules/homebridge-signalk/node_modules/request/request.js:1154:10)
at Request.emit (events.js:315:20)
at IncomingMessage.<anonymous> (/homebridge/node_modules/homebridge-signalk/node_modules/request/request.js:1076:12)
[12/10/2020, 11:23:35 AM] Got SIGTERM, shutting down Homebridge...
[12/10/2020, 11:23:40 AM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null

```

**Comment by commanderpyle** [comment]
This is my config:


{
    "platform": "SignalK",
    "name": "Marutea",
    "host": "127.0.0.1:3000",
    "ssl": false
}

**Comment by Krillle** [comment]
Ok, I see the problem: I didn't catch the error if there is no ignoredPaths section in the config.
As a workaround for now add this line to your config.json: `"ignoredPaths": []`

```
{
"platform": "SignalK",
"name": "Marutea",
"host": "127.0.0.1:3000",
"ssl": false,
"ignoredPaths": []
}
```

**Comment by Krillle** [comment]
Fixed with commit #8
